### ISSUE TITLE
release:npm command is wrong?

### ISSUE THREAD

**Issue by StefH** [mattlewis92/generator-angular-library#38]
This line
```
"release:npm": "npm run build:dist && npm run copyfiles && cd dist/;npm publish",
```

should be:
```
"release:npm": "npm run build:dist && npm run copyfiles && cd dist/ && npm publish",
```

**Comment by mattlewis92** [comment]
The semi colon is intentional and prevents the npm script from changing the working directory to the `dist` folder.

**Comment by StefH** [comment]
On a windows 10 system : This fails.

**Comment by mattlewis92** [comment]
Ah ok, will be fixed in the next release!

**Comment by StefH** [comment]
Can't you just do 
` && npm publish dist`

**Comment by mattlewis92** [comment]
Oh yeah that's a much better way, released as 2.5.3 😄
